src/css/main.scss

```diff
@@ -0,0 +1,94 @@
+@import "../css/body";
+body {
+  background-color: #f1f1f1;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 0 20px;
+  .name {
+    font-size: 40px;
+    font-family: Georgia, "Times New Roman", Times, serif;
+    text-align: center;
+  }
+  p {
+    text-align: center;
+    font-family: Georgia, "Times New Roman", Times, serif;
+  }
+  .text-wrapper {
+    width: 840px;
+    margin: auto;
+  }
+  textarea {
+    outline: none;
+    width: 820px;
+    padding: 10px;
+    border-radius: 10px;
+    border: 2px solid rgb(141, 134, 134);
+    font-size: 20px;
+    font-family: Georgia, "Times New Roman", Times, serif;
+    background-color: #f1f1f1;
+  }
+}
+
+.keyboard-list {
+  padding: 10px;
+  width: 860px;
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  .backspace {
+    width: 125px;
+  }
+  .tab {
+    width: 60px;
+  }
+  .capslock {
+    width: 106px;
+  }
+  .del {
+    width: 58px;
+  }
+  .enter {
+    width: 124px;
+  }
+  .shift {
+    width: 115px;
+  }
+  .space {
+    width: 376px;
+  }
+  .ctrl {
+    width: 60px;
+  }
+  .win {
+    width: 58px;
+  }
+}
+
+.keyboard-button {
+  min-width: 50px;
+  min-height: 50px;
+  padding: 10px;
+  font-size: 20px;
+  margin: 3px;
+  background-color: #474747;
+  color: #fff;
+  border-radius: 5px;
+  transition: 0.3s ease-in-out;
+  box-shadow: 1px 2px 1px 1px #000;
+  &:hover {
+    background-color: rgb(129, 185, 140);
+    border: rgb(129, 185, 140);
+  }
+  &:active {
+    transform: scale(0.95);
+  }
+}
+
+.active {
+  background-color: rgb(129, 185, 140);
+  border: rgb(129, 185, 140);
+  transform: scale(0.95);
+}
```

Comment:
General comment: please check normalize and reset css approaches for more consistent and expected browser behavior in terms of styling. Use the one you choose for this project.

Reply:
Using normalize